Update README.md

JonathanDHolden opened the pull request.

JonathanDHolden committed:
Update README.md

finnlewis commented:
Hi @JonathanDHolden 

Thanks for this, but not sure what we can do with this pull request as it doesn't seem to fix the problem. 

Looking at the commit, I guess there is a broken link that needs updating, probably to https://docs.localgovdrupal.org/devs/workflows/installing-and-deploying-lgd.html 

Would you like to amend the pull request to fix the link or shall we close it? 

Also, we need to write better guidelines for creating pull requests so they are easily testable. But ideally:

1.  the commit message should detail what is being done, rather than just which file is being updated. For example: "Remove broken link."
2. The pull request should confirm steps to review / test / confirm the change has had the desired effect. 

I'll see if I can write something today.

In the mean time, let me know if I can help you with this one.

Many thanks,

Finn

markconroy commented on docs/src/devs/workflows/README.md:
```diff
@@ -28,7 +28,7 @@ Useful backgound reading:
 
 For an example on how to install LocalGov Drupal and deploy changes to a development site:
- -  [Installing and deploying a LocalGov Drupal site](/workflows/installing-and-deploying-lgd)
+ -  [Installing and deploying a LocalGov Drupal site] DEAD LINK
```
Let's set that link to here: https://docs.localgovdrupal.org/devs/getting-started/ instead

aaronhirtenstein committed:
Update README.md

updated link on line 31 to https://docs.localgovdrupal.org/devs/getting-started/
